api_tests/e2e/rfc003/btc_eth/test.js

```diff
@@ -24,90 +28,94 @@ const alpha_max_fee = 5000; // Max 5000 satoshis fee
 describe("RFC003: Bitcoin for Ether", () => {
     before(async function() {
         this.timeout(5000);
-        await test_lib.btc_activate_segwit();
-        await bob.wallet.fund_eth(bob_initial_eth);
-        await alice.wallet.fund_eth(alice_initial_eth);
-        await alice.wallet.fund_btc(10);
-        await test_lib.btc_import_address(bob_final_address); // Watch only import
-        await test_lib.btc_import_address(alice.wallet.btc_identity().address); // Watch only import
-        await test_lib.btc_import_address(bob.wallet.btc_identity().address); // Watch only import
-        await test_lib.btc_generate();
-
-        await test_lib.log_eth_balance(
+        await bitcoin_rpc_client_conf.btc_activate_segwit();
+        await bob.wallet.eth().fund(bob_initial_eth);
+        await alice.wallet.eth().fund(alice_initial_eth);
+        await alice.wallet.btc().fund(10);
+        await bitcoin_rpc_client_conf.btc_import_address(bob_final_address); // Watch only import
+        await bitcoin_rpc_client_conf.btc_import_address(
+            alice.wallet.btc().identity().address
+        ); // Watch only import
```

Comment:
I know it's due to the formatting but could you move the comment to it's own line above the import?